Added click-plugins to requirements.txt

snorfalorpagus opened the pull request:
Test fix for #593.

sgillies commented:
👋 @snorfalorpagus I was dropping a link to https://travis-ci.org/Toblerity/Fiona/jobs/389935268#L1096 and seeing that we have some error or log messages corrupting the pure JSON we expect in one test. It looks like you're on it!

snorfalorpagus commented:
This PR fixes the following issues:

* click-plugins added to requirements.txt
* enum34 added to requirements.txt for Python 2.7 and earlier
* fixed the problem with fixtures in ZipArchiveReadingTest

This closes #593 and is ready for merge. I've removed the attempt at fixing cURL+HTTP from this branch - I'll create a separate ticket for that.

sgillies commented:
@snorfalorpagus our build failures were transient!

Trying to authenticate with AWS unnecessarily is part of the problem and I'm going to work on that before our release.
